docker-rpmbuild.sh

```diff
@@ -1,19 +1,28 @@
 #!/bin/bash
+set -e
+cd $(dirname $0)/install
+if [ -e pkg ]; then 
+  mkdir pkg
+fi
 
-SLURM22="22.05.3"
-SLURM20="20.11.9"
+TMP_BINS=slurm-pkgs-tmp
+rm -rf $TMP_BINS
+mkdir $TMP_BINS
 
-rm -rf blobs-build-tmp
-mkdir blobs-build-tmp
 if command -v docker; then
+  extra_args=
+  if command -v podman; then
```

Comment:
should this be `elif` instead? but since this is inside the outer if loop i am not sure what the purpose is here. If docker is not installed in the system, this won't work anyways. for reference i used the script with podman, and I had to comment out parts of it it to get it to work. We could do something like `if command -v podman; then $runtime=podman` and then use $runtime variable in line 18,19,20. upto you though!